cypress/support/commands/application.js

```diff
@@ -43,3 +43,42 @@ Cypress.Commands.add(
     );
   },
 );
+Cypress.Commands.add(
+  'testResponseCodeAndTypeRetry',
+  (path, responseCode, type, retriesLeft = 2) => {
+    cy.request({ url: path, retryOnStatusCodeFailure: true }).then(
+      ({ status, headers }) => {
+        expect(status, `Unexpected status code for ${path}`).to.equal(
+          responseCode,
+        );
+        expect(
+          headers['content-type'],
+          `Unexpected content-type for ${path}`,
+        ).to.include(type);
+
+        // Ensure we're not seeing the Mozart fallback during smoke testing
+        if (Cypress.env('SMOKE')) {
+          try {
+            expect(
+              headers,
+              `Mozart fallback response detected for ${path}`,
+            ).not.to.have.property('x-mfa');
+          } catch (e) {
+            if (retriesLeft < 1) {
```

Comment:
This is a good idea :)  Trying to understand why `testResponseCodeAndTypeRetry` is defined twice though - ie, it's defined in this file, but also in `cypress/support/commands.js`

Reply:
Yeah I'm not really sure why testResponseCodeAndType existed twice. It says '// These are deliberately duplicated from the other cypress folder to avoid interrelationships blocking changes.' in the commands.js folder, so I did the same for this command as it is almost the same. I'm not sure what that reasoning in the comment means though. I can try removing one and see if it breaks.